data/error_policies/virtual_function_non_define.cas

```diff
@@ -2,4 +2,4 @@ virtual resource foo_parent {
 	virtual fn foo_func(domain source) {}
 }
 
-virtual resource foo inherits foo_parent {} // Illegal non-inheritance
+resource foo inherits foo_parent {} // Illegal non-inheritance
```

Comment:
This is related to the above discussion.  Automatic inheritance of functions outside of resource association isn't implemented yet.  I'm planning to implement that along with the @derive annotation.  Once I do that, the "virtual" version of this would compile successfully (and I'll add a positive test for it then)